media/css/externalpages/pocket/components/_platform-footer.scss

```diff
@@ -0,0 +1,124 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.platform-footer {
+    border-top: 1px solid #cacaca;
+    background-color: $color-light-gray-05;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    padding-top: 32px;
```

Comment:
I realize now that we should probably stick to pre-defined tokens from Protocol if we're able to. We can use `$spacing-xl` here 🪐

Reply:
Good lookin out